app/Http/Controllers/Api/V1/Project/ProjectController.php

```
@@ -125,19 +125,20 @@
     public function updateStage(UpdateProjectStageRequest $request, Project $project)
     {
         $valid = $request->validated();
-        $newStage = ProjectStage::from($valid['stage']);
-        $oldStage = $project->marketing_stage;
+        $oldStage = $project->marketing_stage->value;
+        $newStage = $valid['stage'];
+        $newStageEnum = ProjectStage::validateTransition($oldStage, $newStage);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Enums\ProjectStage' in method 'updateStage'. (Clean Code Rules)

(StaticAccess)